extensions/mssql/src/contracts.ts

```diff
@@ -1151,6 +1151,36 @@ export namespace SqlMigrationRefreshPerfDataCollectionRequest {
 	export const type = new RequestType<SqlMigrationRefreshPerfDataCollectionParams, mssql.RefreshPerfDataCollectionResult, void, void>('migration/refreshperfdatacollection');
 }
 
+export interface StartLoginMigrationsParams {
+	sourceConnectionString: string;
+	targetConnectionString: string;
+	loginList: string[];
+	aadDomainName: string;
```

Comment:
isn't aadDomainName part of the login name?  I would expected it to be in loginList as:
'redmond\abc',
'redmond\defg',
'redmond\etc',

Reply:
as per conversations with Sreraman and Kavitha, it is a bit more involved then just the "redmond" part. apparently, the code should work without an AAD domain name (since user passing that is a security risk) as long as the federation access is set up correctly. Kavitha and I will work on testing that scenario.